src/split-criterions/VarianceReduction.jl

```diff
@@ -9,7 +9,7 @@ Calculate the sample variance of a given set of labels. It uses the standard for
 ## Returns
 - `Float64`: The sample variance of the input label vector `y`.
 """
-function variance(y::AbstractVector)
+function calculate_variance(y::AbstractVector)
     variance = sum((y .- mean(y)) .^ 2) / length(y) - 1
```

Comment:
I already changed this in code-coverage branch but I think this has to be:
variance = sum((y .- mean(y)) .^ 2) / (length(y) - 1) 
otherwise it will be divided by the length(y) first and then substracted 1